src/interpolations.jl

```diff
@@ -249,6 +249,20 @@ match the vertex enumeration of the corresponding geometrical cell.
 """
 vertexdof_indices(ip::Interpolation) = ntuple(_ -> (), nvertices(ip))
 
+"""
+    dirichlet_vertexdof_indices(ip::Interpolation)
+
+A tuple containing tuples of local dof indices for the respective vertex in local
+enumeration on a cell defined by [`vertices(::Cell)`](@ref). The vertex enumeration must
+match the vertex enumeration of the corresponding geometrical cell.
+Used internally in [`ConstraintHandler`](@ref) and defaults to [`vertexdof_indices(ip::Interpolation)`](@ref) for continuous interpolation.
+
+!!! note
+    The dofs appearing in the tuple must be continuous and increasing! The first dof must be
+    the 1, as vertex dofs are enumerated first.
```

Comment:
Should this repeat `vertexdof_indices(ip::Interpolation)` or just refer to it?
I repeated it so it appears in the LSP/IDE tip

Reply:
I mean the resulting docstring should be a bit different, right? There is overlap because we use the same convention, which (as you already do) should be documented.